index.html

```diff
@@ -99,9 +99,18 @@ <h1>About me</h1>
     </p>
 
     <form action="https://formspree.io/f/xjvleokg" method="POST" id="contact-me">
-      <input type="text" placeholder="Full Name" maxlength="30" required>
-      <input type="email" placeholder="Email Address" required>
-      <textarea name="textarea" placeholder="Enter text here" maxlength="500" required></textarea>
+      <div class="field">
+        <input type="text" name="full_name" id="full_name" placeholder="Full Name" maxlength="30">
+        <small></small>
+      </div>
+      <div class="field">
+        <input type="email" name="email" placeholder="Email Address">
+        <small></small>
+      </div>
+      <div class="field" >
+        <textarea name="textarea" placeholder="Enter text here" maxlength="500"></textarea>
+        <small ></small>
+      </div>
       <button type="submit">Get in touch</button>
 
```

Comment:
- [ ] Nice job with validation 👍  but please, consider placing the error message close to submitting button as it is required in the project: `If the validation is not OK, you show an error message to the user near the submit button informing them of the error and the form is not sent.` I know you can do it easily 👍